core/src/banking_stage.rs

```diff
@@ -1424,9 +1330,6 @@ impl BankingStage {
         banking_stage_stats
             .process_packets_count
             .fetch_add(packet_count, Ordering::Relaxed);
-        banking_stage_stats
```

Comment:
Can you also update the stats to be something new like `receive_packets_count` and `receive_packets_elapsed`

Reply:
done